programmers/난이도별/level01.정수_내림차순으로_배치하기/SSSOy.java

```diff
@@ -0,0 +1,16 @@
+import java.util.*;
+
+class Solution {
+    public long solution(long n) {
+        String answer = "";
+        
+        String s = Long.toString(n);
+        String[] arr = s.split("");
+        Arrays.sort(arr, Collections.reverseOrder());
+        
+        for(String a : arr) 
+            answer += a;
+        
+        return Long.parseLong(answer);
```

Comment:
```suggestion
        return Arrays.stream(Long.toString(n).split(""))
                     .map(Integer::parseInt)
                     .sorted((a, b) -> b - a)
                     .map(String::valueOf)
                     .collect(collectingAndThen(joining(""), Long::parseLong));
```
이렇게 stream을 사용하면 더 직관적으로 풀이할 수 있습니다!

Reply:
추가로 `Collectors.*`를 import 해야합니다!
```java
import static java.util.stream.Collectors.*;
```

Reply:
와 알려주셔서 감사합니다!!